tests/test_live.sh

```
@@ -0,0 +1,132 @@
+
+npx mcporter call "${SERVER_NAME}.unraid" \
+  --http-url "$MCP_URL" --header "$AUTH_HEADER" \
+  action=live subaction=cpu > /dev/null 2>&1 \
+  && pass "action/live/cpu" || fail "action/live/cpu" "call failed"
+
+# ── Resources (server-level, no tool name needed) ────────────────────────────
+header "Resources"
+
+npx mcporter call "${SERVER_NAME}" --http-url "$MCP_URL" --header "$AUTH_HEADER" \
+  --list-resources > /dev/null 2>&1 \
+  && pass "resources/list" || skip "resources/list" "no resources defined"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)